Rakefile

```diff
@@ -5,7 +5,24 @@ require 'bundler/gem_tasks'
 
 Bundler::GemHelper.install_tasks
 
-require "hammer_cli_foreman_leapp/version"
-require "hammer_cli_foreman_leapp/i18n"
-require "hammer_cli/i18n/find_task"
+Rake::TestTask.new do |t|
+  t.libs.push 'lib'
+  t.verbose = true
+end
```

Comment:
Why was this needed?

Reply:
How else would you define a Rake task for running the tests?

Reply:
There aren't any tests so the task is not needed. I think this repository should stick with just using Rubocop for now, until tests are written.